port/unix/omrsysinfo.c

```diff
@@ -5416,6 +5417,27 @@ isCgroupV1Available(struct OMRPortLibrary *portLibrary)
 	return result;
 }
 
+/**
+ * @internal
+ * Checks if cgroup v2 system is available
+ *
+ * @param[in] portLibrary pointer to OMRPortLibrary
+ *
+ * @return TRUE if cgroup v2 system is available, FALSE otherwise
+ */
+static BOOLEAN
+isCgroupV2Available(struct OMRPortLibrary *portLibrary)
+{
+	BOOLEAN result = FALSE;
+
+	/* If the cgroup.controllers file exists at the root cgroup, then v2 is available */
+	if (0 == access(OMR_CGROUP_V1_MOUNT_POINT "/cgroup.controllers", F_OK)) {
```

Comment:
`OMR_CGROUP_V1_MOUNT_POINT` should be renamed if it is used with both cgroup v1 and v2.
```suggestion
	if (0 == access(OMR_CGROUP_MOUNT_POINT "/cgroup.controllers", F_OK)) {
```